src/util.h

```diff
@@ -248,6 +248,11 @@ class ArgsManager
      */
     std::string GetChainName() const;
 
+    /**
+     * Clear available arguments
+     */
+    void ClearArgs() { m_available_args.clear(); }
```

Comment:
Should this be `inline`?

Reply:
It doesn't need to be not inline and this is simpler IMO

Reply:
Implementation should be *.cpp

Reply:
@Empact why?